src/BenchmarkDotNet/Templates/BenchmarkProgram.txt

```diff
@@ -1,31 +1,16 @@
 $ShadowCopyDefines$
 $ExtraDefines$
-using System;
-using System.Diagnostics;
-using System.Linq; // must not be removed, used by SmartParameter and SmartArgument
-using System.Threading;
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using BenchmarkDotNet;
-using BenchmarkDotNet.Characteristics;
-using BenchmarkDotNet.Environments;
-using BenchmarkDotNet.Jobs;
-using BenchmarkDotNet.Extensions;
-using BenchmarkDotNet.Helpers;
-using BenchmarkDotNet.Running;
-using BenchmarkDotNet.Engines;
-using BenchmarkDotNet.Validators;
-$Namespaces$
-
 $AdditionalLogic$
+// this file must not be importing any namespaces
```

Comment:
@adamsitnik this condition can be easily checked with the help of a unit test: we can load this file and check that it doesn't contain any `using` declarations.

Reply:
@adamsitnik I created an issue for that: #1010

Reply:
@AndreyAkinshin good idea, will do it